purchase_requisition_incoterm/tests/TEST_INSTRUCTIONS.rst

```diff
@@ -0,0 +1,4 @@
+- Open contact for Deco Addict, and in the Sales&Purchase tab set the default incoterm
+- Open Purchase > Orders > Blanket Orders
+- Under the Other Information tab the default incoterm can be seen
+- Confirm the order, the Purchase Order will have the default incoterm set
```

Comment:
Which incoterm will be set on the purchase.order? The one from the contact or the one from the requisition?

Check with the test if the requisition overrules the the partner.

Reply:
The incoterm cannot be set in the requisition and overrule the partner, it can only be set on the partner record and is carried over the purchase requisition, RFQ and purchase order. I updated the test instructions to make it more clear